content/docs/develop/getting-started/creating-an-addon.md

```diff
@@ -7,6 +7,15 @@ aliases:
 Required software: text editor, Chrome.  
 If possible, disable the Scratch Addons extension you downloaded from the store before proceeding to avoid issues.
 
+
+{{< admonition error >}}

```

Comment:
What's with the error? Why not info?

```suggestion
{{< admonition info>}}
```

Reply:
Sure. Just wanted to use red for some reason.

Reply:
Part of me wants to give a meaning of these colors.